packages/cli/src/Prompt.ts

```diff
@@ -478,9 +491,8 @@ export declare namespace All {
  * @since 1.0.0
  * @category collecting & elements
  */
-export const all: <
-  const Arg extends Iterable<Prompt<any>>
->(arg: Arg) => All.Return<Arg> = InternalPrompt.all
```

Comment:
RE "Supports **multiple arguments**" in jsdocs, AFAIK it only supports a single argument

Reply:
That's true - only an array or a record. I can update the JSDoc and add the example from the changelog as a separate PR.